src/pages/calendar/components/Schedule.tsx

```diff
@@ -34,7 +36,17 @@ export function Schedule({ meetingTimes }: ScheduleProps): JSX.Element {
             <Td>{m.time}</Td>
             {/* TODO: verify if we can safely exclude this for most cases */}
             {/* <Td>{m.scheduleType}</Td> */}
-            <Td>{m.where}</Td>
+            <Td>
+              {isMobile ? (
```

Comment:
Since this logic is repeated a bit, might be good to isolate it in it's own component.

Something like
```jsx
<Location short={`${m.buildingAbbreviation} ${m.roomNumber}`} long={m.where} />
```
For example. Could even have a prop like `alwaysShort` or something for the Event so it never uses the long form.

Reply:
that's a good point!! I just pushed some commits that should make that change